config/initializers/json.rb

```diff
@@ -0,0 +1,6 @@
+# https://github.com/ohler55/oj/blob/develop/pages/JsonGem.md
+# https://github.com/ohler55/oj/blob/develop/pages/Rails.md
+
+require 'oj'
+Oj.optimize_rails
+Oj.add_to_json
```

Comment:
This effectively tells Rails to pipe `as_json` and `to_json` through `Oj` in most scenarios.